### PR TITLE
Update logs metrics

### LINKED ISSUE #1492: 'Accessing logs' needs updates
Serving no longer writes Istio access logs and instructions under 'Accessing request logs' is no longer valid and needs to be replaced with the new access logs we introduced in queue-proxy. In addition, access logs in queue-proxy is disabled by default and we need to document how to enable them in this documentation as well.

### PR DESCRIPTION
**Pull request by yanweiguo**
Fixes #1492

### CODE REVIEW

File: docs/serving/accessing-logs.md [hunk 1]
```diff
@@ -50,12 +50,32 @@ To find the logs sent to `stdout/stderr` from your application in the Kibana UI:
 
 ### Accessing request logs
 
-To find the request logs of your application in the Kibana UI :
+To access the request logs(if enabled), enter the following search in Kibana:
```
Comment (samodell) [inline docs/serving/accessing-logs.md]:
```suggestion
To access the request logs (if enabled), enter the following search in Kibana:
```

File: docs/serving/installing-logging-metrics-traces.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ sections to do so now.
...
 ## Metrics
 
+1. Run the following command to enable request metrics if they are wanted:
+
+   ```
+   kubectl edit cm -n knative-serving config-observability
+   ```
+
+   Add `metrics.request-metrics-backend-destination: prometheus` to `data` field.
+   You can find detail information in `data._example`.
```
Comment (samodell) [inline docs/serving/installing-logging-metrics-traces.md]:
```suggestion
   You can find detailed information in `data._example`.
```

Comment (samodell) [reply docs/serving/installing-logging-metrics-traces.md]: Where is data._example?

File: docs/serving/installing-logging-metrics-traces.md [hunk 3]
```diff
@@ -17,6 +17,15 @@ sections to do so now.
 
 ## Metrics
 
+1. Run the following command to enable request metrics if they are wanted:
```
Comment (samodell) [inline docs/serving/installing-logging-metrics-traces.md]: If you run this command, aren't you opening an editor to update the configs manually? IE, running this command doesn't automatically enable anything; you have to follow the instructions below to enable metrics? If that's the case, I think we should say that.

File: docs/serving/accessing-logs.md (hunk 1 above)
Comment (yanweiguo) [reply docs/serving/accessing-logs.md]: Done.

File: docs/serving/installing-logging-metrics-traces.md (hunk 2 above)
Comment (yanweiguo) [reply docs/serving/installing-logging-metrics-traces.md]: Done.

File: docs/serving/installing-logging-metrics-traces.md (hunk 3 above)
Comment (yanweiguo) [reply docs/serving/installing-logging-metrics-traces.md]: Done.

**Pull request merged**